.github/workflows/unitpacker-go.yml

```
@@ -0,0 +1,255 @@
+# You can request an app-specific password for your Apple ID user at https://appleid.apple.com
+###
+# MAC_STORE_TEAM_ID - Team ID of the user or organization
+# This is the same 10 characters that are displayed in your certificate CN at the end in brackets.
+###
+# GPG_PRIVATE_KEY - PGP private key block for Linux signing
+# GPG_PASSPHRASE - passphrase for the GPG_PRIVATE_KEY
+##
+
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
+# > Note: An event will not be created when you push more than three tags at once.
+
+name: unitpacker-go
+on:
+  push:
+    tags:
+      - "multisig-v*"
+
+jobs:
+
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      config: ${{ steps.set-config.outputs.config }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: set-config
+        run: |
+          if [ -n "${ACT+x}" ]; then
+            ./compile-unitpacker-config.sh gaiad-v99.0.1
+          else
+            ./compile-unitpacker-config-tools.sh "$GITHUB_REF_NAME"
+          fi
+
+  config-test:
+    needs: config
+    strategy:
+      matrix: ${{fromJSON(needs.config.outputs.config)}}
+    runs-on: ${{matrix.os}}
+    steps:
+      - run: |
+          echo "hello world"
+          echo "Tool name: ${{matrix.tools.name}}"
+          echo "Version: ${{matrix.tools.ref}}"
+          echo "Git repo: ${{matrix.tools.git_url}}"
+
+  build:
+    needs: config
+    strategy:
+      matrix: ${{ fromJSON(needs.config.outputs.config) }}
+    runs-on: ${{matrix.os}}
+    steps:
+
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ matrix.tools.git_url }}
+          ref: ${{ matrix.tools.ref }}
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.19.0'
+
+      - name: Install binary
+        run: go install
+
+      - name: Print binary version
+        run: ~/go/bin/${{ matrix.tools.name }} --version
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ${{ github.run_id }}.${{ matrix.os }}
+          if-no-files-found: error
+          retention-days: 3
+          path: ~/go/bin/${{ matrix.tools.name }}
+
+
+  sign:
+    strategy:
+      matrix: ${{ fromJSON(needs.config.outputs.config) }}
+    runs-on: ${{matrix.os}}
+    needs:
+      - config
+      - build
+    steps:
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: ${{ github.run_id }}.${{ matrix.os }}
+
+      - uses: apple-actions/import-codesign-certs@v1
+        if: runner.os == 'macOS'
+        with:
+          p12-file-base64: ${{ secrets.MAC_APP_CERTIFICATE }}
+          p12-password: ${{ secrets.MAC_APP_CERTIFICATE_PASSPHRASE }}
+
+      - name: Sign macOS binary
+        if: runner.os == 'macOS'
+        run: codesign --timestamp -o runtime --sign "${{ secrets.MAC_APP_CERTIFICATE_CN }}" -v ./${{ matrix.tools.name }}
+
+      - name: Verify macOS signature
+        if: runner.os == 'macOS'
+        run: | ##
```

Are the comments necessary there? Just asking...

```suggestion
        run: |
```

---

No necessary. Was a typo. Thanks